app/build.gradle

```diff
@@ -28,6 +28,9 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+    testOptions {
+        animationsDisabled = true
```

Comment:
Is this for performance or do tests not work without this?

Reply:
It's recommended in the Espresso docs to [avoid flakiness](https://developer.android.com/training/testing/espresso/setup#set-up-environment)